wmn-data.json

```diff
@@ -4778,6 +4778,19 @@
         "known" : ["john", "daniel"],
         "cat" : "shopping"
        },
+       {
+        "name" : "Showup.tv",
+        "uri_check" : "https://showup.tv/profile/{account}",
+        "e_code" : 200,
+        "e_string" : "O mnie",
+        "m_string" : "<title>Darmowe",
+        "m_code" : 404,
+        "known" : ["LunaVee", "Jane_Frou"],
+        "cat" : "xx NSFW xx",
+        "headers": {
+          "Cookie": "accept_rules=true;"
+        }
+       },

```

Comment:
Looks like there is an extra `}` here or in the line above. Can you check please?

Reply:
First `}` closing headers dict, second `}` closing site object. 

Json file is valid and 

This is output from github checks:
'✅ All files were validated successfully.'

Reply:
Agreed. TY